cli/cmd/channel_release_demote.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/replicatedhq/replicated/client"
+	"github.com/spf13/cobra"
+)
+
+func (r *runners) InitChannelReleaseDemote(parent *cobra.Command) {
+	cmd := &cobra.Command{
+		Use:   "demote CHANNEL_ID_OR_NAME",
+		Short: "Demote a release from a channel",
+		Long: `Demote a release from a channel.
+
+  Example:
```

Comment:
This should go into the `Example` key in order to render correctly

Reply:
doh 😶‍🌫️ , i was copy/pasting from `enable-semantic-versioning` , will update.